add dbs_to_discover config option to target specific dbs in discover

seanbyrne88 opened the pull request:
@psantacl Per our discussion today, this is the way i came up with to target specific dbs on discover.

psantacl commented:
Sorry.  What I mean is that if the user provides a filter_dbs paramter it should be a comma separated list of database names.  Then, the tap can split the property on commas and get a vector of database names.  This vector of database names can then be fed directly in the SQL here: https://github.com/seanbyrne88/tap-postgres/blob/1c2afada3fd66a2aaa6bad8a01726f6dca32d44f/tap_postgres/__init__.py#L273.


This is an important step i believe because it will prevent the tap from trying to discover DB's that do not exist.  DB's that the user erroneously entered into the config.

seanbyrne88 commented:
Thanks for clarifying. Makes sense to check if the dbs exist. 

Changed to use the comma separated string, although I did like using a JSON array as it reduced complexity a little. There is an extra `split(',')` and I added a `trim` since sometimes a comma separated list ends up using `, `.
